blueprints/examples/src/tests.rs

```diff
@@ -0,0 +1,136 @@
+use crate::eigen_context;
+use crate::eigen_context::ExampleTaskManager;
+use alloy_provider::Provider;
+use blueprint_test_utils::helpers::get_receipt;
+use blueprint_test_utils::incredible_squaring_helpers::start_default_anvil_testnet;
+use blueprint_test_utils::{inject_test_keys, KeyGenType};
+use gadget_io::SupportedChains;
+use gadget_sdk::config::protocol::EigenlayerContractAddresses;
+use gadget_sdk::config::ContextConfig;
+use gadget_sdk::info;
+use gadget_sdk::logging::setup_log;
+use gadget_sdk::runners::eigenlayer::EigenlayerConfig;
+use gadget_sdk::runners::BlueprintRunner;
+use gadget_sdk::utils::evm::get_provider_http;
+use reqwest::Url;
+use std::path::Path;
+use std::time::Duration;
+use tokio::time::timeout;
+
+#[tokio::test]
+async fn test_eigenlayer_context() {
+    setup_log();
+
+    let (_container, http_endpoint, ws_endpoint) = start_default_anvil_testnet(false).await;
+    let url = Url::parse(&http_endpoint).unwrap();
+
+    let provider = get_provider_http(&http_endpoint);
+    info!("Fetching accounts");
+    let accounts = provider.get_accounts().await.unwrap();
+
+    let owner_address = accounts[1];
+    let task_generator_address = accounts[4];
+
+    info!("Deploying Example Task Manager...");
+    let context_example_task_manager = ExampleTaskManager::deploy_builder(provider.clone());
+    let context_example_task_manager_address = get_receipt(context_example_task_manager)
+        .await
+        .unwrap()
+        .contract_address
+        .unwrap();
+    std::env::set_var(
+        "EXAMPLE_TASK_MANAGER_ADDRESS",
+        context_example_task_manager_address.to_string(),
+    );
+    let context_example_task_manager =
+        ExampleTaskManager::new(context_example_task_manager_address, provider.clone());
+
+    info!("Starting Eigenlayer Blueprint Context Test...");
+
+    // Set up Task Spawner
+    let _task_spawner_handle = tokio::task::spawn(async move {
```

Comment:
If this is not being used can you remove it? I had assumed we are using the task spawner from the `incredible_squaring_helpers`

Reply:
The task spawner for the incredible squaring blueprint is different and utilizes the Incredible Squaring Task Manager. This is just a very simple version for the example blueprint.